environments/common/inventory/group_vars/all/openondemand.yml

```diff
@@ -5,7 +5,12 @@
 
 # NB: Variables prefixed ood_ are all from https://github.com/OSC/ood-ansible
 
-# openondemand_servername: '' # Must be defined when using openondemand
+openondemand_servername: "{{ hostvars[groups['openondemand'].0].ansible_host if groups['openondemand'] else '' }}"
+
+openondemand_auth: basic_pam
+
+openondemand_jupyter_partition: "{{ openhpc_slurm_partitions[0]['name'] }}"
```

Comment:
Does this work OK for ordering? E.g. if you try

`ansible all -m debug -a var=openondemand_jupyter_partition` does it work, or does it complain something is undefined? I have a memory of the lexicographical ordering  of openondemand.yml vs openhpc.yml being important here.